autopilot/autopilot.go

```diff
@@ -175,6 +176,34 @@ func (ap *Autopilot) Handler() http.Handler {
 	})
 }
 
+func (ap *Autopilot) writeResponse(c jape.Context, code int, resp any) {
```

Comment:
`code` is unused by the way

Reply:
hey I think @n8maninger  wrote this method not sure what it does I just use it